### PR TITLE
applets/places-indicator: Display all volumes/style & code improvements

### PR DESCRIPTION
**Pull request by cybre**

### CODE REVIEW

File: panel/applets/places-indicator/PlaceItem.vala [hunk 1]
```diff
@@ -14,15 +14,20 @@ public class PlaceItem : ListItem
...
         try {
             GLib.FileInfo info = file.query_info("standard::symbolic-icon", GLib.FileQueryInfoFlags.NONE, null);
             set_button(file.get_basename(), get_icon(info.get_symbolic_icon()));
         } catch (GLib.Error e) {
             set_button(file.get_basename(), get_icon(null));
-            message(e.message);
+            warning(e.message);
         }
+
+        name_button.set_tooltip_text(_("Open \"%s\"".printf(file.get_basename())));
```
Comment (cybre) [inline panel/applets/places-indicator/PlaceItem.vala]: Wouldn't that show as `Open "%s"` in trasifex? Would just `Open` be better?

Comment (ikeydoherty) [inline panel/applets/places-indicator/PlaceItem.vala]: Honestly don't know. What do others do here.. ?

Comment (cybre) [inline panel/applets/places-indicator/PlaceItem.vala]: New commit should work.

**Comment by ikeydoherty** [comment]
Doesn't need further review. It's not Ubuntu's indicator and that's every good thing I need to know.

**Pull request merged**